docs/feature_leader_key.md

```diff
@@ -70,3 +71,69 @@ SEQ_THREE_KEYS(KC_C, KC_C, KC_C) {
   SEND_STRING("Per key timing is great!!!");
 }
 ```
+
+## Customization 
+
+The Leader Key feature has some additional customization that you can perform here.  It has two functions that can be called at certain parts of the process.  Namely `leader_start()` and `leader_end()`.
+
+The `leader_start()` function is called when you tap the `KC_LEAD` key, and the `leader_end()` function is called when either the leader sequence is completed, or the leader timeout is hit. 
+
+You can add these functions to your `keymap.c` file to add feedback (such as beeping or playing music).
```

Comment:
This point is important. Is the keymap.c the only place ? Or there are other places like your user space that can be used too ?
Making this clear is very important IMO because one of the root causes of people having problems with the leader key was that it was not clear that you must define the leader configs on your config.h file

Reply:
For this part, I think it's best to leave it as "keymap".  Or at least refer to "the users' code". 

Generally, we don't talk about the userspace, unless it's the userspace document.  But yes, anything that you could put in your keymap.c, you can put in your userspace.